benchmark-framework/src/main/java/io/openmessaging/benchmark/WorkloadGenerator.java

```diff
@@ -159,9 +160,13 @@ private void ensureTopicsAreReady() throws IOException {
         // In this case we just publish 1 message and then wait for consumers to receive the data
         worker.probeProducers();
 
-        while (true) {
-            CountersStats stats = worker.getCountersStats();
+	// do not loop forever
```

Comment:
What is the reason for this change? Also, if the call to this method is now commented.

Reply:
I made this change first as I was getting stuck in an infinite loop waiting for the probeProducers. It looks like often, but not always the Probe must already have been consumed.

Reply:
what happens if the countdown ends ?

what about reverting this part ?

Reply:
I reverted this part too.